legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/domain/api/review/ReviewApi.java

```diff
@@ -204,6 +215,11 @@ default List<Review> getReviews(Set<ProjectType> projectTypes, boolean assignedT
      */
     Review editReview(String projectId, String reviewId, String title, String description, List<String> labels);
 
+    /**
+     * Returns the predicate for the given workspace Id
+     */
+    Predicate<String> getWorkspaceIdPredicate(String workspaceId);
+
```

Comment:
This method does not belong here. It is up to callers to create any predicate they might want.

Reply:
I also think passing `workspaceId` and `workspaceType` separately is not ideal, they together make the meaningful identifier, so maybe the predicate to be passed into the methods above should encompass both of these.

Reply:
We could do BiPredicate<String, WorkspaceType> which takes both the workspace id and workspace type. That would be the most general.